hyperspy/_signals/eds.py

```diff
@@ -122,7 +122,7 @@ def _get_beam_energy(self):
         units_name = self.axes_manager.signal_axes[0].units
 
         if units_name == 'eV':
-            beam_energy = beam_energy * 1000
+            beam_energy *= 1000
         return beam_energy
```

Comment:
That's true. In this particular case it is not relevant, but I will go through and check